.cargo/config

```diff
@@ -1,11 +1,15 @@
 [target.armv7-unknown-linux-gnueabihf]
 linker = "arm-linux-gnueabihf-gcc"
+pre-build = ["apt-get update && apt-get install -y lld"]
```

Comment:
I think that's a field in Cross.toml, not here?

Reply:
Oh you are right.
But anyway, I think this will not work as it requires `arm-linux-gnueabihf-gcc`.

Reply:
Actually, perhaps it's time to use `zig-cc` for cross compilation instead of `cross` or manually install the compiler as we have done on `aarch64-unknown-linux-gnu`.

Reply:
Worth a try

Reply:
`zig-cc` also uses `clang` and `lld`, so everything here is going to work just fine.

Reply:
> Worth a try

I will probably use [`cargo-zigbuild`](https://github.com/rust-cross/cargo-zigbuild) for this.

Reply:
Ah the known issues on there are all things we've hit here against here, so it may not work

Reply:
Looks like I would have to create a ticket for it.